views/ingress_weighting.erb

```diff
@@ -0,0 +1,26 @@
+<% if list.any? %>
+  <div class="table-responsive">
+    <table class="table">
+  <thead class="thead-dark">
+    <tr>
+      <th scope="col">Namespace</th>
+      <th scope="col">Host</th>
+    </tr>
+  </thead>
+  <tbody>
+  <% list.each do |service| %>
+    <% namespace = service["namespace"] %>
```

Comment:
The namespace variable is not used anywhere. so dont need this line

Reply:
Good spot.

Reply:
21e2907